packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/generalized-react-app.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Template file for new grouping rules. To use:
+ *
+ * 1. Copy TEMPLATE.js to a new grouping rule file.
+ * 2. Change `ChangeMeGroupingRule` class name to another name.
+ * 3. Change `this.name` from `CHANGE_ME` in the constructor.
+ * 4. Fill in the `choose` function.
+ * 5. Include the new rule file in index.js.
+ * 6. Access in views via `this.experiments.choose('name from 3')`
+ *    or `this.isInExperimentGroup('name from 3', 'group name')`.
+ */
+ 'use strict';
+
+ const BaseGroupingRule = require('./base');
+
+ const GROUPS = [
+   'control',
+
+   // Treatment branches. This one is for users who will see the new, generalized React app which houses more urls than just `/settings`
+   'generalized',
```

Comment:
Just a nitpick, if you name this `react` instead of `generalized` that might read _slightly_ nicer for QA and others to see `forceExperimentGroup=react` as a kind of cue that they're seeing the React version.

Reply:
I see what you mean -- I thought that might be more confusing since both of the different versions show React apps. Since this experiment is specifically meant to wrap the generalization of the React app (as opposed to moving specific views into React), this made more sense to me, but I'm open to changing it!

Reply:
Yes makes sense! Great question -- my thinking is as follows: many routes have to be rolled out in batches (for example, all of the reset_password routes), so we need to wrap the entire batch in an experiment. So I was thinking that we would have THIS experiment only wrap around the React app generalization, and we could gradually release it (and then remove it) before adding on any significant routes

Reply:
Ah ok, so just to confirm, this is a temporary experiment to help us develop the first routes that we'll remove once we have the first actual experiment for batched routes in place? Can we add a comment in the experiment file about this and/or in the PR description? 🙂

Reply:
Yes, fantastic idea! Thank you!